api/v1/types/model_types.go

```diff
@@ -31,3 +31,9 @@ type MetricResult struct {
 type MetricResultList struct {
 	Items []MetricResult `json:"items"`
 }
+
+type ExternalEntityListEntry struct {
```

Comment:
I'm not sure that `Point` is descriptive of what this struct represents, as `Point` sounds like a single point in time of a timeseries.

Essentially this struct could be represent any of the following sets of values:
(Node_name, latest_CPU, latest_Mem) 
(Pod_name, latest_CPU, latest_Mem) 
(Namespace_name, latest_CPU, latest_Mem)
(Container_name, latest_CPU, latest_Mem)

I was using the word `Entity` to describe `Node | Pod | Namespace | Container`, which are essentially the entities of the cluster that are represented in the model. Do you think we should look for a better word to describe these "entities"?